src/js/bellows.js

```diff
@@ -176,7 +176,7 @@
         close: function($item) {
             $item = this._item($item);
 
-            if (!$item.hasClass(cssClasses.OPENED)) {
+            if (!$item.hasClass(cssClasses.OPENED) || $item.hasClass(cssClasses.DISABLED)) {
```

Comment:
A good option if there's any more usage for it, but at the moment I think the logic is trivial enough to not need any more abstraction.